python/pyspark/ml/regression.py

```diff
@@ -743,6 +743,18 @@ def treeWeights(self):
         """Return the weights for each tree"""
         return list(self._call_java("javaTreeWeights"))
 
+    @property
```

Comment:
Looking through this, I was just wondering why `featureImportances` is not in `TreeEnsembleModels` instead of a method in each concrete class for the RF and GBT models? @sethah